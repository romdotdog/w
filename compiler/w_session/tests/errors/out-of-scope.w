fn main(): u32 {
    {
        let a = 1;
    };
    a
}