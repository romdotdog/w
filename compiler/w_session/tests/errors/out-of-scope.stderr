tests/errors/out-of-scope.w:6:1: error: identifier does not exist
}
^
tests/errors/out-of-scope.w:6:1: error: type mismatch
}
^
